.github/workflows/pushes.yaml

```diff
@@ -94,6 +94,12 @@ jobs:
     uses: ./.github/workflows/managers.yaml
     secrets: inherit
 
+  build-ivts:
+    name: Build the 'ivts' module
```

Comment:
I guess the built artifacts are just saved as artifacts, rather than pushed into the github maven repo ?

Reply:
The IVTs workflow will download the other Galasa artifacts that were just built and use those